CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Proper syntax highlighting for expressions with metadata.
 - Add basic support for dynamic indentation via `clojure-ts-get-indent-function`.
 - Add support for nested indentation rules.
+- Properly highlight function name in `letfn` form.
```

Comment:
Can you also add the PR number here? And in some separate PR/commit - the PRs for the previous few changes, as I somehow forgot about those.

Reply:
Sure, I usually put links to issues here, but some changes were done without them.